README.md

```diff
@@ -1,2 +1,349 @@
+- Authentication service utilizing [OAuth2 for REST APIs](https://oauth.net/2/) and [DIF DID AuthN](https://identity.foundation/working-groups/authentication.html) for DIF DIDcomm messaging
+- An implementation of the [W3C Universal Wallet](https://w3c-ccg.github.io/universal-wallet-interop-spec/) for key and document management
+- An implementation of the W3C Verifiable Credential Issuance and Verification standard both as REST APIs and as DIDcomm endpoints
+- Implementation of ITN Data Sharing APIs both as REST APIs and as DIDcomm endpoints
+- An implementation of ITN Core Services functionality via DIF DIDcomm
+- An implementation of the [W3C/DIF Encrypted Data Vault](https://identity.foundation/confidential-storage/) for document storage
+
+ITN Agent Integration can be achieved through an SDK, as utilized by the ITN.
+
+![ITN Agent](./docs/images/ITN%20Agent.jpg)
+
+ITN Agent is an implementation of the [Aries Agent RFC](https://github.com/hyperledger/aries-rfcs/blob/main/concepts/0004-agents/README.md)
+with a flexible plugin architecture.
+
+Depending on which plugins and [DIDComm protocols](https://didcomm.org/search/?page=1) you use, your instance of ITN Agent can
+perform a variety of roles:
+
+- Create and manage cryptographic keys
+- Create and manage Decentralized Identifiers
+- Issue and verify Verifiable Credentials (VCs) and Presentations (VPs)
+- Communicate with other Agents over [DIDComm](https://identity.foundation/didcomm-messaging/spec/#discover-features-protocol-20)
+
+Agent provides a common interface for plugins to expand its functionality. It is designed to be modular and extensible,
+so you can add new protocols, transports, and managers to fit your needs.
+
+## ITN SDK
+
+![ITN_SDK](https://github.com/itn-trust/itn-did-spec/assets/18353464/6135b073-c085-41fd-8ea4-f80e64e0485d)
+
+The ITN SDK has an Identity Protocol provides methods to manage DIDs.
+The ITN SDK has a DID Document Manager provide a set of functions to manage DID Documents.
+
+## Conformance
+
+The keywords MAY, MUST, MUST NOT, RECOMMENDED, SHOULD, and SHOULD NOT in this document are to be interpreted according to [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119)`
+
+## Terminology
+
+1. ITN - Integrated Trust Network
+2. EDV - Encrypted Data Vault
+3. CAS - Content Addressable Storage
+4. DLT - Distributed Ledger Network
+5. DID - Decentralized Identifier
+6. VC - Verifiable Credentials
+7. VP - Verifiable Presentations
+
+## DID ITN Method Specification
+
+### Format
+
+The ITN DID is derived as follow:
+
+- A random 32 byte string is generated
+- The seed is hashed using sha512
+- An Ed25519 key pair is generated from the hash. The key pair will be used as the recovery key pair
+- The uuid of the did:itn is the base58 encoded value of public keys hexadecimal value
```

Comment:
The identifier of the `did:itn` is the first 16 bytes of the public key encoded into Base58 format.

Reply:
@AlexMesser are you asking to add that point or update the last point (line 103)?

Reply:
to update line 103

Reply:
@AlexMesser updated as per your request. Resolving the comment.